cdk/apps/dev_worker/README.md

```diff
@@ -60,7 +66,7 @@ Context parameters are defined with the `-c "STACK_NAME=my-stack` argument.
 Repeat the `-c` parameter for defining multiple parameters.    
 i.e 
 ```
-cdk deploy -c "STACK_NAME=my-stack"
+cdk deploy -c "STACK_NAME=my-stack" -c "USE_SPOT_INSTANCE=false"
```

Comment:
@alexiswl Why? SPOT instances are **very** much preferred, by default :)

Reply:
Happy for it to be configurable, as long as the default is SPOT.
However, I would remove the spot price parameter. It's kind of annoying to first find out what the current spot price is and then adjust it each time you want to change the instance type.

Reply:
Can't pause a spot instance and every now and again they go away which can be annoying. I think for the dev worker use case there's definitely a need for non-spot instances.

Reply:
A default spot price parameter needs to be set though right?

Reply:
You don't need a spot price parameter (IIRC), only setting the `MarketType` to `spot` should use the default spot price (which is usually the on-demand price).

Reply:
So one should never need to set the spot price? Or should this remain as an option?

Reply:
I would vote for the former. The less effort the better and setting a spot price explicitly is only useful in very rare cases.